server/pruning.go

```diff
@@ -36,3 +43,71 @@ func GetPruningOptionsFromFlags(appOpts types.AppOptions) (pruningtypes.PruningO
 		return pruningtypes.PruningOptions{}, fmt.Errorf("unknown pruning strategy %s", strategy)
 	}
 }
+
+// PruningCmd prunes the sdk root multi store history versions based on the pruning options
+// specified by command flags.
+func PruningCmd(providerCreator types.StoreProviderCreator) *cobra.Command {
```

Comment:
Can you add this cmd to simapp.

Reply:
Wait, why simapp? IMO this is a general command and should really probably exist under `client/`. Thoughts?

Reply:
I put `PruningCmd` in `server/pruning.go` because it is similar to `StartCmd`.
I think it is ok to move this `PruningCmd` to `client/pruning` if you think it's more appropriate

Reply:
> Wait, why simapp? IMO this is a general command and should really probably exist under `client/`. Thoughts?

I mean add to the root cmd of simapp.
Regarding the package name, I agree `client` would be better since it has nothing to do with the "server".

Reply:
`PruningCmd` is moved to `client/pruning` and added to the root cmd of simapp.